src/main/java/com/netcracker/study/objects/borders/Wall.java

```diff
@@ -12,6 +12,20 @@ public Wall(int thickness) {
         this.thickness = thickness;
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
have you generated equals method by IDEA? If yes then you can resolve this issue if not then please generate it

Reply:
replaced "equals" with IDEA-generated method